### ISSUE TITLE
if searchbox filter returns single result, hitting enter should select it

### ISSUE THREAD

**Issue by ara4n** [vector-im/riot-web#3658]

**Comment by lampholder** [comment]
Yep. Do we have a single searchbox component that it's easy to update with this behaviour?

**Comment by t3chguy** [comment]
if we had a single Searchable List maybe, but a Searchbox knows not what it gives its string to, say a Room List is more complex than a single list of entries

**Comment by t3chguy** [comment]
This is already implemented for the Room List but not for Room Directory / Member List
